### DISCUSSION TITLE
GPS update rate 10hz vs 5hz

### DISCUSSION THREAD

**Discussion by Jetrell** [iNavFlight/inav#8741] [General]
@stronnag @breadoven 
I've got a question to run by you guys.
Over the last few months I've started using the Matek M10 GPS on my smaller quads. Mostly due to the excessive 1500Mhz noise some HD VTX's emit in their data processing.
The M10 has a considerably better LNA and SAW filter.. Being able to see through the noise floor between -2 to -4dbm lower than the M8.
The main difference I see between the two, is the update rate. 

Now comes my observation.  When comparing two quads of the same size (3"), in Poshold... One with an M8Q Ublox7 10hz and the other with an M10 Ublox 5hz.
The 10hz quad is rock solid in position hold. Even with a low satellite count of 10 and a HDOP of 1.1.
While the 5hz quad is always jittery on the roll and pitch axis. trying to hold position, even with a satellite count of 30 and a HDOP of 0.8.
 I've tried every thing to tune its Nav settings, even with the VTX turned off, with no improvement.. But when I had an M8Q on this quad. It was not jittery in Poshold.

 _Do you think a slower refresh rate could be effecting position hold accuracy in this way?_

I don't see this effect when testing with Ardupilot.. But that isn't really conclusive. Because it uses an EKF for better sensor fusion filtering.

Thanks

**Comment by mmosca** [comment]
@Jetrell maybe you can test #9103 

That will set m10 to 10Hz if set to UBLOX7.

**Comment by mmosca** [comment]
And just for completeness, it may limit the max number of satellites the gps will be able to track (but I have seen at least 18 with m10@10Hz)
